navsat_transform:
  broadcast_utm_transform: true
  delay: 0.0
  frequency: 4
  magnetic_declination_radians: 0.18494
  publish_filtered_gps: true
  use_odometry_yaw: false
  wait_for_datum: false
  yaw_offset: 0.0
  zero_altitude: true
